_config.php

```diff
@@ -4,8 +4,10 @@
 use SilverStripe\Core\Object;
 use MadeHQ\Markdown\Forms\MarkdownEditorField;
 
-define('MARKDOWN_MODULE_BASE', basename(dirname(__FILE__)));
+define('MARKDOWN_MODULE_BASE', 'mademedia/silverstripe-markdown');
```

Comment:
Just a minor question on this, what's the reasoning behind altering this to be hard coded, over just leaving it to be dynamically set in PHP?

Reply:
I think `basename(dirname(__FILE__))` will only return the current folder (eg: silverstripe-markdown), in SS4 you need to provide the composer package name when trying to require JS/CSS.

As you guys are using `MARKDOWN_MODULE_BASE` in all your `Requirements` calls, it was a little easier to just change this.

I am not sure if there is a way of getting the package name from composer direct...